### DISCUSSION TITLE
ORAN 7.2 Split: Multiple DUs and RUs

### DISCUSSION THREAD

**Discussion by rolandmartinson7890** [srsran/srsRAN_Project#168] [General Help] [answered]
I would like to ask if it is possible, with the current version of srsRAN Project, to connect multiple DUs to a single CU and/or multiple RUs to a single DUs using the ORAN switch. 

Will I be able to get metrics like CQI, MCS, RSRP, SINR from each of the RUs connected (even if I am using a single DU)?

**Comment by mrouili** [comment]
Bumping this as I am wondering the same thing.

**Comment by andrepuschmann** [comment, marked as answer]
Hey,
yes it's possible to instantiate multiple cells in a single gNB. Under the hood this is realized through multiple DUs, each providing a single cell. The feature isn't fully announced but will be part of the next official release also introducing mobility.

**Reply by rolandmartinson7890** [reply]
Thanks for your reply @andrepuschmann. When can we expect the next release and by "mobility" you mean HandOvers will now be supported?

**Comment by andrepuschmann** [comment]
Yes, by "mobility" I mean handover. The official release will be around April I suppose. But as you might have noticed we are pushing code out much more often.
Are you at a stage were you'd like to test with multiple RUs yourself or you're just asking?

**Reply by rolandmartinson7890** [reply]
That is awesome. No, I just wondered. I would probably use multiple RUs by Q4 2024 so an April release works for me. Good to know what is ahead and thanks for your work!
